.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -2,15 +2,16 @@
   <profile version="1.0">
     <option name="myName" value="Project Default" />
     <inspection_tool class="AndroidLintNewerVersionAvailable" enabled="true" level="WARNING" enabled_by_default="true" />
-    <inspection_tool class="AutoCloseableResource" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="AutoCloseableResource" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="METHOD_MATCHER_CONFIG" value="java.util.Formatter,format,java.io.Writer,append,com.google.common.base.Preconditions,checkNotNull,org.hibernate.Session,close,java.io.PrintWriter,printf,android.database.sqlite.SQLiteOpenHelper,getWritableDatabase|getReadableDatabase,javax.inject.Provider,get" />
+    </inspection_tool>
```

Comment:
Are the changes to this file intended?

Reply:
No, I think those lines are added automatically.